### DISCUSSION TITLE
How to fit for nginx rewrite

### DISCUSSION THREAD

**Discussion by extrame** [vuepress/core#798] [Q&A] [answered]
Hello guys,

I hold my site behind nginx using vuepress.

I try to make some link as /doc/kind-a/123.html to /doc/kind-a.html?id=123(which share a same md file kind-a.md).

Nginx success render the kind-a.html, but after a while seems that vuepress check some error happen so redirect to 404 page.

Why? and how to fix?

**Comment by Mister-Hope** [comment, marked as answer]
Reproduction repo and website required.
